components/kern-table/KernTable.tsx

```diff
@@ -0,0 +1,133 @@
+import { Fragment } from "react";
+import { IconEdit } from "@tabler/icons-react";
+import KernDropdown from "../KernDropdown";
+import { NotApplicableBadge } from "@/submodules/react-components/components/Badges";
+import { Tooltip } from "@nextui-org/react";
+import MultilineTooltipAutoContent from "@/submodules/react-components/components/MultilineTooltipAuto";
+
+export default function KernTable(props: KernTableProps) {
+    return (
+        <table className="min-w-full divide-y divide-gray-300 rounded-b-lg">
+            <thead className="bg-gray-50">
+                <tr>
+                    {props.headers.map((header) => (
+                        <th scope="col"
+                            className={`px-3 py-2 text-center text-xs font-medium uppercase tracking-wide text-gray-500 ${header.hasSort ? 'hover:text-gray-700 cursor-pointer' : ''}`}
+                            id={header.id} key={header.id}
+                            onClick={header.hasSort ? () => props.config.onClickSort(header.id) : undefined}
+                        >
+                            {header.hasCheckboxes ? <>
+                                <input
+                                    type="checkbox"
+                                    className="h-4 w-4 rounded border-gray-300 text-indigo-600 focus:ring-indigo-600 cursor-pointer"
+                                    checked={header.checked}
+                                    onChange={header.onChange}
+                                />
+                            </> : <div className="inline-flex flex-row items-center">
+                                {!header.tooltip ? header.column :
+                                    <div className="flex w-full justify-center">
+                                        <Tooltip
+                                            content={<MultilineTooltipAutoContent tooltip="True positives / (True positives + False positives)\nfor the reference data you labeled" splitOn="\n" />}
+                                            color="invert"
+                                            placement="top">Est. Precision</Tooltip>
+                                    </div>}
+                                {header.hasSort && <SortArrows sortKey={props.config.sortKey} property={header.id} />}
+                            </div>}
+                        </th>))}
+                </tr>
+            </thead>
+            <tbody className="divide-y divide-gray-200 bg-white">
+                {props.values?.map((row, index) => (
```

Comment:
i think in some tables with have "no content yet" or similar message.
Example cognition api table 
![image](https://github.com/user-attachments/assets/924b7355-3034-46b9-8357-9dcf2913efce)


could make sense to include here/as parameter